roles/confluent.common/tasks/gather_config_logs.yml

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
how about "gather_logs.yml" or "fetch_logs.yml"? and use gather or fetch in the playbook filename as well for consistency